lib/papers/manifest_command.rb

```diff
@@ -0,0 +1,53 @@
+module Papers
+  class ManifestCommand
+    def initialize(manifest_path = nil)
+      @manifest_path = manifest_path || File.join('config','papers_manifest.yml')
+    end
+
+    def manifest_exists?
+      !!File.exist?(@manifest_path)
```

Comment:
`File.exist?` will only ever return `true` or `false`. You shouldn't need the `!!`